plugins/workbench-resources/src/components/SavedView.svelte

```diff
@@ -85,13 +96,52 @@
     ]
   }
 
+  async function switchPublicAction (object: FilteredView, originalEvent: MouseEvent | undefined): Promise<Action[]> {
+    return [
+      {
+        icon: object.sharable ? task.icon.TodoCheck : task.icon.TodoUnCheck,
+        label: view.string.PublicView,
+        action: async (ctx: any, evt: Event) => {
+          await client.update(object, { sharable: !object.sharable })
```

Comment:
What happens if I unshare shared view that someone already uses?

Reply:
Hm, interesting one, need to check on already saved ones.

Reply:
View made non-public is left as is for all users already added it, but becomes not visible in view search for adding.